src/async-components/views/dialogs/secretstorage/CreateSecretStorageDialog.js

```diff
@@ -527,6 +552,8 @@ export default class CreateSecretStorageDialog extends React.PureComponent {
 
     _titleForPhase(phase) {
         switch (phase) {
+            case PHASE_RESTORE_KEY_BACKUP:
+                return _t('Restore Your Key Backup');
```

Comment:
Maybe change casing to:

```suggestion
                return _t('Restore your Key Backup');
```

We also seem unsure whether it should be "Key Backup" or "key backup" (we've used both equally), but I'll leave that battle for another time.